package.json

```diff
@@ -35,6 +36,7 @@
   "scripts": {
     "fix": "npm run lint -- --fix",
     "lint": "eslint .",
+    "makeCheckpoints": "node -r sucrase/register scripts/makeCheckpoints.ts",
```

Comment:
As mentioned in the task, we are adding an `update-sources` script to the repo as part of the SDK upgrade. So, we should probably name this guy `update-checkpoints` to match the naming scheme. The name of the .ts file doesn't matter.

Otherwise, we would want to change the SDK upgrade PR, and react-native-mymonero as well, since those wouldn't match.